pkg/artifactory/resource/replication/resource_artifactory_local_repository_multi_replication.go

```diff
@@ -0,0 +1,375 @@
+	},
+	"enabled": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     true,
+		Description: "When set, enables replication of this repository to the target specified in `url` attribute. Default value is `true`.",
+	},
+	"include_path_prefix_pattern": {
+		Type:             schema.TypeString,
+		Optional:         true,
+		ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
+		Description:      "List of artifact patterns to include when evaluating artifact requests in the form of x/y/**/z/*. When used, only artifacts matching one of the include patterns are served. By default, all artifacts are included (**/*).",
+	},
+	"exclude_path_prefix_pattern": {
+		Type:             schema.TypeString,
+		Optional:         true,
+		ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
+		Description:      "List of artifact patterns to exclude when evaluating artifact requests, in the form of x/y/**/z/*. By default, no artifacts are excluded.",
+	},
+	"proxy": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Proxy key from Artifactory Proxies settings. The proxy configuration will be used when communicating with the remote instance.",
+	},
+	"replication_key": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Computed:    true,
+		Description: "Replication ID. The ID is known only after the replication is created, for this reason it's `Computed` and can not be set by the user in HCL.",
+	},
+	"check_binary_existence_in_filestore": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "Enabling the `check_binary_existence_in_filestore` flag requires an Enterprise Plus license. When true, enables distributed checksum storage. For more information, see [Optimizing Repository Replication with Checksum-Based Storage](https://www.jfrog.com/confluence/display/JFROG/Repository+Replication#RepositoryReplication-OptimizingRepositoryReplicationUsingStorageLevelSynchronizationOptions).",
+	},
+}
+
+var localMultiReplicationSchema = map[string]*schema.Schema{
+	"repo_key": {
+		Type:             schema.TypeString,
+		Required:         true,
+		ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
+		Description:      "Repository name.",
+	},
+	"cron_exp": {
+		Type:             schema.TypeString,
+		Required:         true,
+		ValidateDiagFunc: validator.CronLength,
+		Description:      "Cron expression to control the operation frequency.",
+	},
+	"enable_event_replication": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "When set, each event will trigger replication of the artifacts changed in this event. This can be any type of event on artifact, e.g. add, deleted or property change. Default value is `false`.",
+	},
+	"replications": {
+		Type:     schema.TypeList,
+		Optional: true,
```

Comment:
Should this constraint to `MinItem: 1`? A replication config must have at least one target, right?

Reply:
Yes, good idea. The user can't save the replication without at least one item there.